OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -82,6 +82,11 @@ public interface IProductionIconOverlay
 		bool IsOverlayActive(ActorInfo ai);
 	}
 
+	public interface INotifyInsufficientFunds
+	{
+		void InsufficientFunds();
```

Comment:
This should accept an `Actor self` parameter for consistency with other interfaces.